sanic worker doc update (#7864)
RasaHQ/rasa: docs/docs/http-api.mdx

for more details). This will only work in combination with the
`RedisLockStore` (see [Lock Stores](./lock-stores.mdx).

:::caution

> Using multiple worker processes does not work when using the 

Reviewer: ```suggestion
The [SocketIO channel](./connectors/your-own-website.mdx#websocket-channel) does not support multiple worker processes. 
```

> [SocketIO channel](https://rasa.com/docs/rasa/reference/rasa/core/channels/socketio). Instead, use multiple 

Reviewer: ```suggestion
```

> containers or pods with a single Sanic worker to scale up.

Reviewer: ```suggestion
```
